CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
       - Fixed an issue that would detect turning circles as sliproads
       - Fixed a bug where post-processing instructions (e.g. left + left -> uturn) could result in false pronunciations
       - Fixes a bug where a bearing range of zero would cause exhaustive graph traversals
+    - Infrastructure:
+      - Adds a feature to limit results in nearest service with a default of 100 in `osrm-routed`
```

Comment:
It's actually a good point: we don't guarantee ABI stability.

@jcoupey I know you started integrating `libosrm` for VROOM. Is this something that could bite you or are you compiling `libosrm` from source anyway? (Or do you expect your users to do so?)